docs/guides/jobs/metrics.md

```diff
@@ -52,7 +52,25 @@ print('{{"metric": "loss", "value": {}}}'.format(loss))
 # {"metric": "loss", "value": 1.000000}
 ```
 
+
 You can send any metric you want as a Training Metric, however the only values we accept currently are `float` or `integer` values. If you are interested in sending other values as custom training metrics, please let us know by sending an email to support@floydhub.com
 
+### Steps
+
+By default, Training Metrics will show `Time` as the x-axis. However, you can also toggle the Training Metrics to display steps, as long as you provide that information in your logs.
+
+Here is the basic format. Notice that you'll need to provide an integer value for the `step` field:
+```python
+print('{"metric": "<metric_name>", "value": <int_or_float>, "step": <int>}')
+```
+
+We will also display `epoch` as a Step if you emit this value. Here's an example:
```

Comment:
This seems like a format and not an example. Can you also include examples (similar to the previous section):

```
print('{"metric": "accuracy", "value": 0.985, "epoch": 3}')
# {"metric": "accuracy", "value": 0.985. "epoch": 3}
```

Reply:
Yes!